docs/section5/Rule5-47.md

```diff
@@ -0,0 +1,53 @@
+
+# Envelope - Rule 5-47  
+
+**Rule ID:** 5-47  
+**Rule Description:** The baseline air leakage rate of the building envelope (I75Pa) at a fixed building pressure differential of 0.3 in. of water shall be 1 cfm/ft2.  The air leakage rate of the building envelope shall be converted to appropriate units for the simulation program using one of the methods in Section G3.1.1.4.  
+**Rule Assertion:** Sum of B-RMR infiltration.infiltration.air_leakage_rate = expected value.  
+**Appendix G Section:** Section G3.1-5(h) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** 
+
+  1. get_surface_conditioning_category()
+  2. get_zone_conditioning_category()
+
+## Rule Logic:  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dict_b = get_surface_conditioning_category(B_RMR)`
+
+- Get zone conditioning category dictionary for B_RMR: `zone_conditioning_category_dict_b = get_zone_conditioning_category(B_RMR)`
+
+- For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`
+
+  - For each thermal block in building segment: `for thermal_block_b in building_segment_b.thermal_blocks:`
+
+    - For each zone in thermal block: `for zone_b in thermal_block_b.zones:`
+
+      - For each surface in zone: `for surface_b in zone_b.surfaces:`
+
+        - Check if surface is regulated, add zone total area of building envelope to building total: `if scc_dict_b[surface_b.id] != "UNREGULATED": building_total_envelope_area += sum(surface.area for surface in zone_b.surfaces)`
+
+      - Check if zone is conditioned or semi-heated, add zone air leakage rate to building total: `if zone_conditioning_category_dict_b[zone.id] in [CONDITIONED RESIDENTIAL, CONDITIONED NON-RESIDENTIAL, CONDITIONED MIXED, SEMI-HEATED]: building_total_air_leakage_rate += zone_b.infiltration.air_leakage_rate`
+
+- Calculate the required baseline air leakage rate at 75Pa: `target_air_leakage_rate_b = 1.0 * building_total_envelope_area * 0.00047194745`
+
```

Comment:
@xingcx I'm assuming the 0.00047 number is converting from in wg to Pa. Please add a note somewhere to state the same.

Reply:
@xingcx for additional clarity, we should probably state the unit of the calculated value - baseline infiltration airflow m3/s @ 75 Pa

Reply:
@supriyagoel this is conversion from air leakage rate in CFM (1.0 CFM) to air leakage rate in cubic meters per second.

Reply:
RCT team will be using a library for conversions. It will be based on actual units. Can instead say 'Pa_to_psi'. RCT team will eventually upload a file with the actual conversion factors.

Reply:
if there are value comparisons, then unit conversions don't need to be done in the RDS. However, if the unit conversion is a part of the RDS logic, then the conversion can be shown using terminology (like cubic feet per minute and cubic meter per second) but not values.

Reply:
Per last comment from Supriya, need to replace 0.00047 with (cubic feet per minute to cubic meter per second)

Reply:
Updated.

Reply:
@xingcx and @supriyagoel given the confirmed direction regarding the units, we should remove * CONVERSION("CFM", "M^3/S")` from line 35.

Reply:
@mcneillj Its been a while since we discussed this. How should the units be mentioned here in the RDS. I believe it is ok to mention CONVERSION("CFM", "M^3/S")` - please clarify.

Reply:
Updated. The unit conversion is removed.

Reply:
@xingcx line 35 says 'Calculate the required baseline infiltration flow rate in cfm: `target_infiltration_flow_rate_b = 1.0 * building_total_envelope_area`

i believe total envelope area will be in m2, and 1 is CFM/ft2. 
we should convert the total envelope area to ft2. Probably need to add a line that says building_total_envelope_area = building_total_envelope_area (m2 to ft2)
We don't want to include the actual conversion factors (since the RCT team is using Pint for that) but do want to identify when a conversion needs to take place.

Reply:
as discussed at the last RDS call, there is a default list of units and the unit conversions will only be mentioned if a unit is something other than the units in that list.